packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -124,6 +125,11 @@ export interface DropdownProps extends UIComponentProps<DropdownProps>, Position
      * @param item - Dropdown removed element.
      */
     onRemove?: (item: ShorthandValue<DropdownItemProps>) => string;
+    /**
+     * Callback that creates custom accessibility message about the selected items count a screen reader narrates on input field focus.
+     * @param count - number of items selected.
+     */
+    itemsCount?: (count: number) => string;
```

Comment:
could we add a test for the `itemsCount` function ?

Reply:
I've added a more complete test, please see Dropdown-test.tsx.